src/components/InformationsTable.tsx

```diff
@@ -33,14 +33,14 @@ const InformationsTable = <T extends object>({ rows, data, caption, headerCellWi
   return (
     <Table>
       {caption && <TableCaption>{caption}</TableCaption>}
-      {Array.isArray(data) && (
-        <TableRow>
-          <TableCell className="w-80" />
-          <TableCellHeader>{t('primary')}</TableCellHeader>
-          <TableCellHeader>{t('secondary')}</TableCellHeader>
-        </TableRow>
-      )}
       <TableBody>
+        {Array.isArray(data) && (
```

Comment:
this should be in <TableHead>

Reply:
In case that we need to make pdf when `Array.isArray(data)`  is true we have 2 table bodies and our pdf createTable function make table broken. Are you sure we should have here 2 tables bodies?